widgetssdk/src/main/res/layout/chat_view.xml

```diff
@@ -77,7 +77,7 @@
         android:layout_width="match_parent"
         android:layout_height="1dp"
         android:layout_marginTop="@dimen/glia_medium"
-        android:background="?attr/gliaBaseNormalColor"
+        android:background="?attr/gliaBaseShadeColor"
```

Comment:
Is this intended?

Reply:
Yes, I checked with ios and Figma, it was uncorrect before